core/controllers/profile.py

```diff
@@ -532,6 +532,23 @@ def post(self):
 class SiteLanguageHandler(base.BaseHandler):
     """Changes the preferred system language in the user's preferences."""
 
+    Supported_Languages = [
+        'en', 'pt-br', 'ar', 'hi', 'es', 'bn', 'fr', 'id',
+        'uk', 'sk', 'nl', 'kab', 'vi', 'tr', 'zh-hans', 'zh-hant'
+    ]
+
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'site_language_code': {
+                'schema': {
+                    'type': 'basestring',
+                    'enum': Supported_Languages
```

Comment:
How does this work, I don't remember that we have something like `'enum'`.

Reply:
![enum](https://user-images.githubusercontent.com/78922463/153416995-76ff8f24-6182-4691-b4f9-dacf3dc69019.png)
I have found this while I am searching how we can restrict the values of a particular variable ("**site_language_code**" in our case)  to a list of words.

Reply:
Hey @vojtechjelinek , If you do not have any further queries about this schema, Can you please restart e2e tests as I have found no error when I tested them in my local machine as they might be flaky in this case too like the case of my previous PR?

Reply:
Where did you found this? Here is a wiki for our schema implementation https://github.com/oppia/oppia/wiki/Schemas

Reply:
Yes, I saw this Schemas page but I couldn't understand how we can restrict the values of variable in schema to a list of words with that page. So, I have searched for it in internet and used "enum". Can you please give me an idea of how we have to write schema so that it checks the words present in the given list of languages ?

Reply:
> Please check `choices` in the https://github.com/oppia/oppia/wiki/Schemas or you can look at `validators`.

Thanks a lot @vojtechjelinek I have updated the code with the required changes, Can you please verify the changes ?

Reply:
Hello @vojtechjelinek Can you please let me know if everything is fine and ready to merge or not?

Reply:
Hey @vojtechjelinek It looks like you are swamped with other tasks or maybe my code might have some problems, but I would appreciate it if you can give your review about these changes I have made.